operators/src/source/gdal_source.rs

```diff
@@ -183,14 +186,12 @@ impl GdalDatasetInformation for JsonDatasetInformationProvider {
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct TimeIntervalInformation {
-    pub time_intervals: Vec<TimeInterval>,
+    pub start_time: TimeInstance,
+    pub time_step: TimeStep,
+    // pub end_time: Option<TimeInstance>,
```

Comment:
What about the end time?

Reply:
i dont know yet. i guess we have to discuss this

Reply:
changed to a todo: 84514580db159da3b7928253e6390c6f8fafdb5c